docs/components/concepts/access-control/authorizations.md

```diff
@@ -0,0 +1,150 @@
+
+2. **Owners**
+
+   - Types: users, groups, roles, and mapping rules
+   - Authorizations can be assigned to any owner type
+
+3. **Permissions**
+
+   - Define allowed interactions with resources
+   - Specific to resource types
+
+4. **Resources**
+   - Entities users interact with
+   - Each has specific available permissions
+
+## Configuration
+
+### SaaS Configuration
+
+Authorizations can be activated/deactivated on a per-cluster level by:
+
+- Organization Admins
+- Organization Owners
+
+### Self-Managed Configuration
+
+Enable authorizations in your deployment using:
+
+```yaml
+camunda.security.authorizations.enabled: true
+```
+
+## Resources and Permissions
+
+### Available Resources
+
+The following table lists all resources that support authorization in Camunda 8 Orchestration Cluster, along with the available permissions per resource. This applies to Camunda 8 Orchestration Cluster (Zeebe, Operate, Tasklist, Orchestration Cluster APIs).
+
+| Resource Type                        | Resource Key Example                                    | Supported Permissions                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| **Application**                      | `Operate`, `Tasklist`, `Identity`, `*`                  | `ACCESS`                                                                                                                   |
+| **Authorization**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Batch**                            | `*`                                                     | `CREATE`, `READ`, `DELETE`                                                                                                 |
+| **Decision Definition**              | `*`, `decisionId: order_decision`                       | `CREATE_DECISION_INSTANCE`, `READ_DECISION_DEFINITION`, `READ_DECISION_INSTANCE`, `DELETE_DECISION_INSTANCE`               |
+| **Decision Requirements Definition** | `*`, `decisionRequirementsDefinitionId: order_decision` | `CREATE`, `READ`, `UPDATE`                                                                                                 |
+| **Group**                            | `*`, `groupId: accounting`                              | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Mapping Rules**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
```

Comment:
🔧 Should this get the `mappingRuleId` as a resource id?

Reply:
added mappingruleid